site/docs/v1/tech/admin-guide/webauthn.adoc

```diff
@@ -0,0 +1,137 @@
+
+[.api]
+[field]#Enabled#::
+Enabling this toggle will enable the associated workflow for applications belonging to this tenant. The availability of a workflow can be overridden at the application level using the link:#application-configuration[Application Configuration].
+
+[field]#Authenticator attachment#::
+This setting controls which authenticator attachments are allowed for registration ceremonies when using the associated workflow. See the section link:#authenticator-attachment[below] for recommendations on configuring this option.
+
+[field]#User verification#::
+This setting controls whether user verification is required for registration and authentication ceremonies when using the associated workflow. See the section link:#user-verification[below] for recommendations on configuring this option.
+
+===== Authenticator attachment
+
+There are two link:https://www.w3.org/TR/webauthn-2/#authenticator-attachment-modality[authenticator attachment modalities] in WebAuthn.
+
+ * `platform` - the authenticator is integrated with the client device
+ * `cross-platform` - the authenticator is removable from the client device and can be used with other client devices
+
+The workflow settings allow limiting the associated workflow to one attachment only or allowing either. The value is used to limit eligible authenticators to those that are best suited for the workflow. The authenticator attachment preference only impacts the WebAuthn registration ceremony. These are the recommended values for available WebAuthn workflows:
+
+ * Bootstrap - Cross-platform only. The bootstrap workflow can be used to sign in on a new device using WebAuthn. If the authenticator is integrated with the client device, it cannot be used on another.
+ * Re-authentication - Platform only. In order to ensure that the authenticator is available for repeated logins on the same device, it is best to limit authenticator selection to those integrated with the client device.
+
+:advanced_feature: WebAuthn cross-platform authenticators
+include::docs/v1/tech/shared/_advanced-edition-blurb-api.adoc[]
+
+:advanced_feature!:
+
+===== User verification
+
+There are three options for the link:https://www.w3.org/TR/webauthn-2/#enum-userVerificationRequirement[user verification requirement] defined by WebAuthn.
+
+ * `Required` - user verification is required to successfully complete the WebAuthn ceremony
+ * `Preferred` - user verification is preferred for the WebAuthn ceremony, but it will not fail if user verification is not provided
+ * `Discouraged` - user verification should not be provided for the WebAuthn ceremony
+
+The workflow settings allow selecting the preference for user verification. If the workflow allows a user to authenticate, it is _highly_ recommended to require user verification. If user authentication is not required, anyone with access to the authenticator could use it to authenticate as the user that owns the passkey.
```

Comment:
Are there any workflows that don't allow the user to authenticate?

Reply:
Not right now. We plan to add support for MFA using WebAuthn.

Reply:
okay.